src/app/components/school-menu/school-item.tsx

```diff
@@ -10,28 +10,28 @@ interface SchoolItemProps {
 
 export const SchoolItem = ({ item, index, color }: SchoolItemProps) => {
   return 'description' in item ? (
-    <li key={item.title} className="school-item">
+    <li key={item.title}>
       <Link
         to={item.to}
         onClick={!index ? () => window.scrollTo({ top: 0 }) : undefined}
-        className={color}>
-        {item.title}
+        className="school-item">
+        <span className={color}>{item.title}</span>
+        <small>{item.description}</small>
       </Link>
-      <small>{item.description}</small>
     </li>
   ) : (
-    <li key={item.id} className="school-item with-icon">
-      <div className="icon-wrapper">
-        <img src={item.iconSmall} alt={item.title} width={32} height={32} loading="lazy" />
-      </div>
-      <div className="details">
-        <Link to={item.detailsUrl} className={color}>
-          {item.title}
-        </Link>
-        <small>
-          {new Date(item.startDate) > new Date() ? `Start ${item.startDate}` : 'Upcoming'}
-        </small>
-      </div>
+    <li key={item.id} className="school-item">
+      <Link to={item.detailsUrl} className="with-icon">
+        <div className="icon-wrapper">
+          <img src={item.iconSmall} alt={item.title} width={32} height={32} loading="lazy" />
+        </div>
+        <div className="details">
+          <span className={color}>{item.title}</span>
+          <small>
+            {new Date(item.startDate) > new Date() ? `Start ${item.startDate}` : 'Upcoming'}
```

Comment:
I'm opposed to calculations inside the return statement. I believe that components should display data, not compute it. I suggest calculating a variable in advance.

Reply:
Took the calculations out of the render.